invenio_rdm_records/fixtures/demo.py

```diff
@@ -113,6 +114,22 @@ def fake_language(cls):
         random_id = random.choice(["eng", "aah", "aag"])
         return {"id": random_id}
 
+    @classmethod
+    def fake_creators_role(cls):
```

Comment:
Since it is used for both creator and contributor below... shall we use the mixed name?
```suggestion
    def fake_creatibutor_role(cls):
```